test/throw.test.js

```diff
@@ -68,13 +67,13 @@ test('Fastify should throw for an invalid schema, printing the error route - bod
     done()
   }, { prefix: 'hello' })
 
-  fastify.ready(err => {
-    t.equal(err.code, 'FST_ERR_SCH_VALIDATION_BUILD')
-    t.match(err.message, /Failed building the validation schema for POST: \/hello\/form/)
+  await fastify.ready().catch(err => {
```

Comment:
we can use `await t.assert.reject(fastify.ready(), (err)=>{...})`

Reply:
`t.assert.reject` does not exist as a method, I think you meant:   `t.assert.rejects`, correct ? 

 but fails the test if you use:  `t.assert.rejects` ( see screenshot )

<img width="1574" alt="screenshot-test-fail" src="https://github.com/user-attachments/assets/2b53e694-935b-4cc3-bcfb-b1690c269292" />

Reply:
This example seems to work for me:

```js
test('Fastify should throw for an invalid schema, printing the error route - headers', async (t) => {
  t.plan(1)

  const badSchema = {
    type: 'object',
    properties: {
      bad: {
        type: 'bad-type'
      }
    }
  }
  const fastify = Fastify()
  fastify.get('/', { schema: { headers: badSchema } }, () => {})
  fastify.get('/not-loaded', { schema: { headers: badSchema } }, () => {})

  await t.assert.rejects(fastify.ready(), {
    code: 'FST_ERR_SCH_VALIDATION_BUILD',
    message: /Failed building the validation schema for GET: \//
  })
})

```

Reply:
Correct! see commit here: https://github.com/fastify/fastify/pull/6002/commits/52d245c80cfe162457abe45408c3d30ad52158d1